src/test/resources/Features/OCD-3161-VerifyShortcutsPagesLoadSuccessfully.feature

```diff
@@ -0,0 +1,33 @@
+@Regression
+Feature: OCD-3161 Navigation to Shortcuts Pages is successful and displays correct page title
+
+Scenario: Verify navigation to API Documentation page is successful and displays correct page title
+    Given I am on API Documetation page
+    Then I see "API Information for 2015 Edition Products" as the page title for the Api Documentation page
+
+Scenario: Verify navigation to Banned Developers page is successful and displays correct page title
+    Given I am on Banned Developers page
+    Then I see "Developers Under Certification Ban" as the page title for the Banned Developers page
+
+Scenario: Verify navigation to Charts page is successful and displays correct page title
+    Given I am on CHPL charts page
+    Then I see "Charts" as the page title for the CHPL Charts page
+
+Scenario: Verify navigation to Decertified Products page is successful and displays correct page title
+    Given I am on Decertified Products page
+    Then I see "Decertified Products" as the page title for the Decertified Products page
+
+Scenario: Verify navigation to Inactive Certificates page is successful and displays correct page title
+    Given I am on Inactive Certificates page
+    Then I see "Inactive Certificates" as the page title for the Inactive Certificates page
+
+Scenario: Verify navigation to Corrective Action Status page is successful and displays correct page title
+    Given I am on Corrective Action Status page
+    Then I see "Products: Corrective Action Status" as the page title for the Corrective Action Status page
+
+Scenario: Verify navigation to SED Info for 2015 Products page is successful and displays correct page title
+    Given I am on SED Info for 2015 Products page
```

Comment:
This step name doesn't match the scenario (nor do the others). If the scenario is testing navigation to the page, then there can't be a "Given" that you're already on the page. If you want to test navigation, then there are two tests to make; one is from a different page, using the dropdown menu, the other is using the URL directly, though I think that's not as important a test as using the navigation. These should be something more like:

> Given a user is on the Overview page
> When they navigate to the "SED Information for 2015 Edition Products" shortcut
> Then they should see "the title, or whatever"

That "When" above should be a single step that passes in the text of the link in the shortcut menu navigation, then it can be used by all of the tests

Oh, and why are there all of these blank lines at the end of the file? I know it's a minor thing, and if I didn't have other comments I wouldn't bother, but it just looks messy